daemon/stop.go

```diff
@@ -48,7 +49,7 @@ func (daemon *Daemon) containerStop(container *container.Container, seconds int)
 
 	// 2. Wait for the process to exit on its own
 	if _, err := container.WaitStop(time.Duration(seconds) * time.Second); err != nil {
-		logrus.Infof("Container %v failed to exit within %d seconds of SIGTERM - using the force", container.ID, seconds)
+		logrus.Infof("Container %v failed to exit within %d seconds of signal %d - using the force", container.ID, seconds, stopSignal)
```

Comment:
`using the force` isn't particularly descriptive or insistent of the gravity of the situation here. I like Star Wars, but can we change that to `force killing` as well?